pkg/capabilities/consensus/ocr3/types/generate.go

```diff
@@ -0,0 +1,2 @@
+//go:generate protoc --go_out=../../../../ --go_opt=paths=source_relative --go-grpc_out=../../../../ --go-grpc_opt=paths=source_relative --proto_path=../../../../ capabilities/consensus/ocr3/types/ocr3_types.proto values/pb/values.proto
```

Comment:
Could this be expressed more simply if it lived somewhere else?

Reply:
Yes, but I think that would require all protos living in the same package -- not necessarily a bad thing, but not the model we're following atm.

Reply:
I just mean the command. For example, would moving this generate line up the tree allow us to shed some of the `../`?

Reply:
Ah yes, it would. If we moved it to the top-level directory that encompassed all of the dependencies, then the .. would go away.

Do you think it's OK to have the commands detached from their physical location? That feels weird to me.

An alternative would be to have a make proto command that we can run rather than using `go generate` everywhere.

Reply:
> Do you think it's OK to have the commands detached from their physical location? That feels weird to me.

I agree, but I guess I'm not really understanding the issue. Why does only one of the existing generate commands have this complexity?
https://github.com/smartcontractkit/chainlink-common/blob/main/pkg/loop/internal/pb/generate.go#L1-L10

Reply:
Ah it's because this generate command depends on the values protos which are defined in another directory. Without adding those protos to the list of files to generate, protoc can't resolve the dependency.

The solution to that is apparently to make sure that protoc is run from a common parent dir, hence the ...

Reply:
BTW we have encountered a similar problem with mercury, see: 

```
// NOTE: the relative paths in the proto_path are to ensure we find common utilities, like BigInt
//go:generate protoc --proto_path=.:../.. --go_out=. --go_opt=paths=source_relative --go-grpc_out=. --go-grpc_opt=paths=source_relative datasource_v1.proto
```

Reply:
One way I can see around this is to add:

```
protoc --go_out=. --go_opt=paths=source_relative --go-grpc_out=. --go-grpc_opt=paths=source_relative --proto_path=. $(find . -type f -name "*.proto")
```

to `make generate`.

With some changes to our import paths (and possibly some further polish), this should automatically find all our protos with no further need to add each file one by file.

Reply:
If you think that dynamic approach is robust then that sounds great.


> The solution to that is apparently to make sure that protoc is run from a common parent dir, hence the dots.

Otherwise, doesn't this suggest that our assumption about having these commands live near the _output_ file is misguided? It seems like everything is smoother if the command is located based on the _input_ files, no?

Reply:
Cool, I'll follow up with this. In the meantime do you mind approving this PR so it's unblocked?